backend/integrationtest/api/testUtil/nahtavillaolo.ts

```diff
@@ -11,7 +11,7 @@ import { expectToMatchSnapshot } from "./util";
 import { loadProjektiFromDatabase, testPublicAccessToProjekti } from "./tests";
 import { UserFixture } from "../../../test/fixture/userFixture";
 
-const { expect } = require("chai");
+const { expect } = require("chai"); //
 
 export async function testNahtavillaOlo(oid: string, projektiPaallikko: string): Promise<void> {
```

Comment:
Tämä on tarpeeton muutos, näiden commitointia voisi mielestäni välttää

Reply:
Ei tarvitse korjata, mutta mainitsin vaan 👍

Reply:
Tuo tuli mergesotkun yhteydessä. Joskus rebase menee jumiin, ja ainoa tapa päästä eteenpäin, on tehdä joku mielivaltainen muutos. Mutta voin jatkossa tehdä vaikka vaan uuden commitin rebasettamisen jälkeen, niin pääsee noista eroon diffissä. Sit vaan tulee tosi mystillisiä "turhia" committeja branchiin.